src/main/java/com/myfjc/text_file_reader/Application.java

```diff
@@ -0,0 +1,165 @@
+package com.myfjc.text_file_reader;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class Application {
+    public static void main(String[] args) throws FileNotFoundException {
+
+        try {
+            File myLyrics = new File("/home/el_barto/IdeaProjects/FJC_Repo/src/main/java/text_file_reader"
+                    + "/data.txt");
+            String[] listOfWords = getListOfWordsFromFile(myLyrics);
+
+            List<Counter> countersOfBadOrShortWords = new ArrayList<>();
+            List<Counter> countersOfGoodWords = new ArrayList<>();
+            int totalNumberOfGoodWords = 0;
+            int totalNumberOfBadWords = 0;
+            for (String word : listOfWords) {
+                boolean badOrShortWord = word.length() < 3 || itsABadWord(word);
+                addNewWordOrIncreaseCount(word, (badOrShortWord)
+                        ? countersOfBadOrShortWords
+                        : countersOfGoodWords);
+                if (!badOrShortWord) {
+                    totalNumberOfGoodWords += 1;
+                } else {
+                    totalNumberOfBadWords += 1;
+                }
+            }
+            System.out.printf("Total number of good words in the analyzed text-file is %d.\n",
+                    totalNumberOfGoodWords);
+            System.out.printf("Total number of bad (or short) words in the analyzed text-file is %d.\n",
+                    totalNumberOfBadWords);
+            /*System.out.println(countersOfBadOrShortWords.toString());*/
+            String beginning = "The list of all bad and short words from this song are:";
+            printFirstNWordsFromList(countersOfBadOrShortWords.size(), countersOfBadOrShortWords, beginning);
+
+            int N = 5;
+             beginning = String.format("\nThe top %d words from this song are:", N);
+            printFirstNWordsFromList(N, countersOfGoodWords, beginning);
+        } catch (FileNotFoundException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    private static String[] getListOfWordsFromFile(File myLyrics) throws FileNotFoundException {
+        Scanner sc = new Scanner(myLyrics);
+        String lyrics = "";
+        while (sc.hasNextLine()) {
+            String data = sc.nextLine();
+            lyrics = lyrics.concat(data + " ");
+
+        }
+        lyrics = lyrics.replaceAll(",", "");
+        lyrics = lyrics.replaceAll("\\.", "");
+        lyrics = lyrics.replaceAll("\\?", "");
+        lyrics = lyrics.replaceAll("!", "");
+        lyrics = lyrics.toLowerCase();
+
+        return lyrics.split(" ");
+    }
+
+    private static boolean itsABadWord(String word) {
+        for (String badWord : getListOfBadWords()) {
+            if (word.equalsIgnoreCase(badWord)) {
+                return true;
+            }
+        }
+        return false;
```

Comment:
you can replase this code with such line
`Arrays.asList(getListOfBadWords()).contains(word.toLowerCase());`

(but if you change method getListOfBadWords() to return list (as it returns array right now))
it even could look like this
`getListOfBadWords().contains(word.toLowerCase();`

Reply:
gotcha